install_config/downgrade.adoc

```diff
@@ -262,40 +240,34 @@ and Restore].
 [[verifying-the-downgrade]]
 == Verifying the Downgrade
 
-To verify the downgrade, first check that all nodes are marked as *Ready*:
-
-====
+. To verify the downgrade, first check that all nodes are marked as *Ready*:
++
 ----
 # oc get nodes
 NAME                        STATUS                     AGE
 master.example.com          Ready,SchedulingDisabled   165d
 node1.example.com           Ready                      165d
 node2.example.com           Ready                      165d
 ----
-====
 
-Then, verify that you are running the expected versions of the *docker-registry*
+. Then, verify that you are running the expected versions of the *docker-registry*
 and *router* images, if deployed:
-
-====
++
 ----
 ifdef::openshift-enterprise[]
 # oc get -n default dc/docker-registry -o json | grep \"image\"
     "image": "openshift3/ose-docker-registry:v3.1.1.6",
 # oc get -n default dc/router -o json | grep \"image\"
     "image": "openshift3/ose-haproxy-router:v3.1.1.6",
```

Comment:
Probably should update these 3.1 image tags to show the latest versions of these images for 3.4 instead.